javarosa/util/schema-gen/src/org/javarosa/engine/XFormEnvironment.java

```diff
@@ -93,8 +94,12 @@ private InstanceInitializationFactory createIIF() {
 
     private EvaluationContext getEC() {
```

Comment:
Its hard for me to figure out what this function is doing without tracking down a bunch of class variables - any way to clean this up? Maybe move the logic determining customDate into a separate function with a descriptive name? or failing that add a comment?

Also - won't this have the effect of always overriding the today() and now() functions?